meson.build

```diff
@@ -0,0 +1,56 @@
+project(
+    'python-rtmidi',
+    'cpp',
+    version: '1.5.0rc1',
+    license: 'MIT',
+    default_options: [
+        'warning_level=2'
+    ],
+    meson_version: '>=0.53.0'
+)
+
+# Dependencies
+cpp = meson.get_compiler('cpp')
+jack2_dep = dependency('jack', version:'>=1.9.11', required: false)
+jack1_dep = dependency('jack', version:'>=0.125.0, <1.0', required: false)
+alsa_dep = dependency('alsa', required: false)
+thread_dep = dependency('threads')
+coremidi_dep = dependency('threads', required: false)
+winmm_dep = dependency('threads', required: false)
```

Comment:
You already have this mandatory once... do you need to find it 3 times?

Reply:
Those were just placeholders until I have figured / tried out the proper dependency checks for macOS and windows.

I've now put in checks for the CoreMIDI framework and `winmm` dependencies, put I'm sure I'll have to make more adjustments after the first CI runs.